app/main.py

```diff
@@ -14,4 +14,4 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    return str.count(phrase.lower(), letter.lower())
```

Comment:
You could also use it like `phrase.lower().count(letter.lower())`, it is more common way